vividus/src/main/java/org/vividus/transformer/CartesianProductTableTransformer.java

```diff
@@ -60,6 +61,11 @@ public String transform(String tableAsString, TableParsers tableParsers, TablePr
         isTrue(repeatingKeys.isEmpty(), "Tables must contain different keys, but found the same keys: %s",
                 repeatingKeys);
 
+        if (hasEmptyTable(tables))
+        {
+            return getEmptyTableWithHeaders(tables, properties);
+        }
```

Comment:
what will happen with empty table(s) without this `if` (if they go through the logic below)?